Sources/UrlFormEncoding/UrlFormDecoder.swift

```diff
@@ -10,21 +10,37 @@ public final class UrlFormDecoder: Decoder {
   public private(set) var codingPath: [CodingKey] = []
   public var dataDecodingStrategy: DataDecodingStrategy = .deferredToData
   public var dateDecodingStrategy: DateDecodingStrategy = .deferredToDate
-  public var parsingStrategy: ParsingStrategy = .accumulatePairs
+  public var parsingStrategy: ParsingStrategy = .accumulateValues
   public let userInfo: [CodingUserInfoKey: Any] = [:]
 
   public init() {
   }
 
   public func decode<T: Decodable>(_ type: T.Type, from data: Data) throws -> T {
-    let container = self.parsingStrategy.strategy(String(decoding: data, as: UTF8.self))
+    let query = String(decoding: data, as: UTF8.self)
+    let container: [String: Any]
+    switch self.parsingStrategy {
+    case .accumulateValues:
```

Comment:
Kinda funny to get here after that struct vs. enum convo last week! The one interesting thing that ya lose when moving from an `enum` to a `struct` that wraps a closure is the ability to tag and switch on the cases!